draft-dns-zone-digest.xml

```diff
@@ -485,37 +485,45 @@ An alternate method (rfc include) is described in the references. --><!ENTITY RF
           </t>
         </section>
 
-        <section title="The Digest Type Field">
+        <section title="The Scheme Field">
           <t>
-            The Digest Type field is an 8-bit unsigned integer
-            that identifies both the cryptographic hash algorithm used to construct the digest,
-            and the methods by which data is presented as input to the hashing function.
+            The Scheme field is an 8-bit unsigned integer that identifies
+            the methods by which data is collated and presented
+            as input to the hashing function.
           </t>
           <t>
-            At the time of this writing, SHA384-SIMPLE, with value 1, is the only
-            standardized
-            Digest Type defined for &RRNAME; records.  The Digest Type registry
+            At the time of this writing, SIMPLE, with value 1, is the only
+            standardized Scheme defined for &RRNAME; records.  The Scheme registry
             is further described in <xref target="iana"/>.
           </t>
           <t>
-            Digest Type values 240-254 are allocated for Private Use as
+            Scheme values 240-254 are allocated for Private Use as
             described in <xref target="RFC8126"/>.
           </t>
         </section>
 
-        <section title="The Parameter Field">
+        <section title="The Hash Algorithm Field">
+          <t>
+            The Hash Algorithm field is an 8-bit unsigned integer
+            that identifies the cryptographic hash algorithm
+            used to construct the digest.
+          </t>
+          <t>
+            At the time of this writing, SHA384, with value 1, is the only
+            standardized
+            Hash Algorithm defined for &RRNAME; records.  The Hash Algorithm registry
+            is further described in <xref target="iana"/>.
+          </t>
           <t>
-            The Parameter field is an 8-bit unsigned integer whose
-            meaning depends on the Digest Type in use.  For SHA384-SIMPLE,
-            the Parameter field plays no role in digest calculation or verification
-            and is always set to zero.
+            Hash Algorithm values 240-254 are allocated for Private Use as
+            described in <xref target="RFC8126"/>.
           </t>
         </section>
 
         <section title="The Digest Field">
           <t>
             The Digest field is a variable-length sequence of octets
-            containing the message digest.  The Digest field must not be empty.
+            containing the digest value.  The Digest field must not be empty.
```

Comment:
I think that "output of the digest function" is cleaner than "digest value", but don't feel strongly.

Reply:
How about "output of the hash algorithm" since that is the term used above this text?

Reply:
WFM